view/frontend/templates/js/replacejs.phtml

```diff
@@ -684,14 +718,29 @@ if (!$block->isOnPageFromWhiteList() && !$block->isMinicartEnabled()) { return;
 
         var boltCheckoutConfigure = function(cart, hints, callback, parameters) {
             var callConfigure = function() {
+                // Check if BoltCheckout is defined (connect.js executed).
+                // If not, postpone processing until it is
+                if (!window.BoltCheckout) {
+                    whenDefined(window, 'BoltCheckout', callConfigure);
+                    return;
+                }
```

Comment:
This is the part that makes sure we have the polyfill loaded to use promises in IE11 right?

Reply:
Yes and lines 865-867 as well